packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -19,6 +19,10 @@
 LATENCY = 120
 
 
+class NotCriticalException(Exception):
```

Comment:
Doesn't seem necessary; see below. We can just return a certain value to retry instead of creating a whole new exception

Reply:
Yeah that does seem better. That's how it was originally from the stuff I refactored but I'll change it.